src/core/ext/resolver/README.md

```diff
@@ -0,0 +1,3 @@
+# Resolver
+
+Implementations of various name resolution schemes.
```

Comment:
Perhaps add a link to doc/naming.md here?

This will be more relevant once #9106 is reviewed -- hint, hint. :)